test/functional/toolchains/cmake/test_cmake_toolchain_win_clang.py

```diff
@@ -97,26 +98,52 @@ def test_clang_mingw(self, client, runtime):
         static_runtime = (runtime == "static")
         check_vs_runtime(cmd, client, "17", build_type="Release", static_runtime=static_runtime)
 
+    @pytest.mark.tool("ninja")
     @pytest.mark.tool("visual_studio", "17")
     @pytest.mark.parametrize("generator", ["Ninja", "NMake Makefiles"])
     def test_clang_cmake_ninja_nmake(self, client, generator):
         client.run("create . --name=pkg --version=0.1 -pr=clang -s compiler.runtime=dynamic "
-                   "-s compiler.runtime_version=v143 "
+                   "-s compiler.runtime_version=v144 "
                    '-c tools.cmake.cmaketoolchain:generator="{}"'.format(generator))
 
         assert 'cmake -G "{}"'.format(generator) in client.out
         assert "GNU-like command-line" in client.out
-        assert "main __clang_major__17" in client.out
-        assert "main _MSC_VER1941" in client.out
+        assert "main __clang_major__18" in client.out
+        assert "main _MSC_VER194" in client.out
         assert "main _MSVC_LANG2014" in client.out
         assert "main _M_X64 defined" in client.out
         assert "main __x86_64__ defined" in client.out
         cmd = re.search(r"MYCMD=(.*)!", str(client.out)).group(1)
         cmd = cmd + ".exe"
         check_vs_runtime(cmd, client, "17", build_type="Release", static_runtime=False)
 
+    @pytest.mark.tool("ninja")
     @pytest.mark.tool("visual_studio", "17")
-    @pytest.mark.tool("clang", "17")  # repeated, for priority over the mingw64 clang
+    def test_clang_cmake_ninja_clang_cl(self, client):
```

Comment:
This test is not really covering the changes above, as those are Autotools/Gnu Toolchain only.
But it is covering the CMake case with clang-cl that was not sufficiently covered before.